packages-experimental/uniui/src/commands/operations/set-editor-resize.operation.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { IOperation } from '@univerjs/core';
+import { CommandType } from '@univerjs/core';
+
+export const SetEditorResizeOperation: IOperation = {
+    id: 'sheet.operation.set-editor-resize',
+    type: CommandType.OPERATION,
+    handler: (accessor, params) => true,
```

Comment:
Do not use operation as events which is an anti-pattern in our codebase. This will cause performance issue since there are lots of command listeners.

Reply:
The code has been removed